### DISCUSSION TITLE
Can't delete workflow run

### DISCUSSION THREAD

**Discussion by anidotnet** [community/community#26059] [Actions]
<p>Hi,</p>
<p>I am getting 500 error page while deleting a workflow run. Here is the workflow run - <a href="https://github.com/nitrite/nitrite-java/actions/runs/157542793" rel="nofollow noopener">https://github.com/nitrite/nitrite-java/actions/runs/157542793</a></p>
<p>Could you please delete this run?</p>

**Comment by cschleiden** [comment, marked as answer]
<p>Could you try again please? We fixed a bug with workflow deletion at the end of last week.</p>

**Comment by anidotnet** [comment]
<p>Thanks. I am able to delete it now.</p>
